src/FM.LiveSwitch.Mux/Session.cs

```diff
@@ -703,74 +703,81 @@ private async Task<bool> MuxVideo(MuxOptions options)
                 }
             }
 
-            var chunkListFileName = $"{ProcessOutputFileName(options.OutputFileName)}_video_chunks.list";
-            var chunkListFile = Path.Combine(GetTempPath(options), chunkListFileName);
-            var chunkListFilePath = Path.GetDirectoryName(chunkListFile);
-            if (!Directory.Exists(chunkListFilePath))
+            if(chunkFiles.Count > 0)
```

Comment:
Should be a space after the `if` for proper formatting.

Reply:
I would change this to:

```csharp
if (chunkFiles.Count == 0)
{
    return true;
}
```

... and then leave the rest of the function as-is to avoid the extra nesting and make it easier to read.